sdk/resourcemanager/azure-resourcemanager-containerservice/src/test/java/com/azure/resourcemanager/containerservice/KubernetesClustersTests.java

```diff
@@ -36,6 +39,11 @@
 
     @Test
     public void canCRUDKubernetesCluster() throws Exception {
+        // enable preview feature of ACR Teleport for AKS
+        Context context = new Context(
+            AddHeadersFromContextPolicy.AZURE_REQUEST_HTTP_HEADERS_KEY,
+            new HttpHeaders().set("EnableACRTeleport", "true"));
+
```

Comment:
Sample code to enable Teleport for AKS (with `.create(context)` and `.apply(context)` below.